rfcs/0000-api-cr-crowding.md

```diff
@@ -0,0 +1,337 @@
+- Feature Name: api_cr_crowding
+- Start Date: 2021-09-23
+- RFC PR: [mbta/technology-docs#0000](https://github.com/mbta/technology-docs/pull/0000)
+- Asana task: [asana link](https://app.asana.com/)
+- Status: Proposed
+
+# Summary
+
+[summary]: #summary
+
+We want to provide predicted commuter rail crowding data for upcoming trips. The data will be
+consumed from a feed provided by Keolis. We will follow the lead of this [static GTFS-Occupanices
+proposal](https://github.com/google/transit/pull/240), but for now the data will only appear in
+the API and not in our GTFS feed. The API will gain a new "occupancies" resource. The data will be
+documented in the API's swagger docs as "Experimental" with a warning that it could change in the
+future. The implementation will be as minimal and simple as possible, but nevertheless built in a
+solid and (hopefully) forward-compatible way. The immediate concrete use case for this feature is
+showing crowding information on dotcom's CR timetables pages.
+
+# Motivation
+
+[motivation]: #motivation
+
+This data feed will allow clients to easily see expected crowding for CR trips. In particular,
+dotcom is planning to add this information to the CR timetable pages.
+
+# Guide-level explanation
+
+[guide-level-explanation]: #guide-level-explanation
+
+<!-- Explain the proposal as if it was already implemented and you were teaching it to a new developer
+that just joined the team. That generally means:
+
+- Introducing new named concepts.
+- Explaining the feature largely in terms of examples.
+- Explaining how programmers should _think_ about the feature, and how it should impact the way they
+  work on this project. It should explain the impact as concretely as possible.
+- If applicable, provide sample error messages, deprecation warnings, or migration guidance.
+- If applicable, describe the differences between teaching this to senior developers and to junior
+  developers.
+-->
+
+This introduces into the API the proposed new GTFS static resource called
+[occupancies.txt](https://github.com/google/transit/pull/240/files), which contains information
+about "usual or expected in-vehicle occupancy levels". The underlying data can apply to either a
+whole trip or individual stop times along the trip. A given occupancy specifies its
+`occupancy_status` which is an enum ranging from `0` for "Empty" to `6` for "Full". Similar to how
+we represent vehicles in the API, the API response will be the enum word rather than the number.
+Our feed will have an _extra_ field, currently not part of the spec, but which is being advocated
+for, `occupancy_percentage`, which is a value from `0.0` to `1.0`. Each occupancy has some
```

Comment:
`occupancy_percentage` in the VehiclePositions feed is an integer,`0` to `100` (or more)